fem/fespace.hpp

```diff
@@ -308,6 +345,17 @@ class FiniteElementSpace
    bool Conforming() const { return mesh->Conforming(); }
    bool Nonconforming() const { return mesh->Nonconforming(); }
```

Comment:
Should `FiniteElementSpace::Conforming()` return `false` if the space is variable order?

Reply:
I'm not sure. The P matrix with the p-interface constraints can be built for conforming meshes too. Then you can start doing nonconforming refinements, the P will be rebuilt and will continue to work. But maybe for the sake of simplicity we could require that the mesh is nonconforming when you want to have a variable order space, to limit the number of combinations there can be (C/NC mesh, constant/variable order space).

Reply:
I agree that doing something like `EnsureNCMesh` when the space is variable order could be a simple solution here.